### DISCUSSION TITLE
Error metric parse error: expected tag at 1:2: "1"

### DISCUSSION THREAD

**Discussion by jaytnw** [jblance/mpp-solar#82] [Q&A]
I get error when start telegraf. It seems can't insert string to InfluxDB . help me please.

[inputs.mqtt_consumer] Error in plugin: metric parse error: expected tag at 1:2: "A"
[inputs.mqtt_consumer] Error in plugin: metric parse error: expected field at 1:7: "True - 1/False - 0"

**Comment by jaytnw** [comment]
When debug use -D

DEBUG:helpers:key_wanted: No filter and key _command not excluded by excl_filter None so key wanted
DEBUG:helpers:key_wanted: No filter and key _command_description not excluded by excl_filter None so key wanted
DEBUG:helpers:key_wanted: No filter and key ac_input_voltage not excluded by excl_filter None so key wanted
DEBUG:helpers:key_wanted: No filter and key ac_input_frequency not excluded by excl_filter None so key wanted
DEBUG:helpers:key_wanted: No filter and key ac_output_voltage not excluded by excl_filter None so key wanted

**Reply by jblance** [reply]
What command and output module is being used to generate the mqtt messages 
Post the full command and debug output
What version is being used?
